server/nimble_server.py

```diff
@@ -19,22 +19,41 @@
 # Allow imports from the parent directory (potential security issue)
 sys.path.append(os.path.dirname(__file__) + "/..")
 
-from generate import generate_docs
+from generate import generate
```

Comment:
At some point we probably want to move the generate functionality into the nimble_orchestration module so we don't have do do weird path things. `generate.py` can then basically become a 2-3 line script.

Reply:
Issue created here: https://github.com/Wakoma/nimble/issues/72